mne/annotations.py

```diff
@@ -0,0 +1,79 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+import json
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float, shape (n_annotations,)
+        Annotation time onsets from the beginning of the recording.
```

Comment:
I think it's better to leave them in their original order. You can always do the sorting / rearranging step within the plotting function itself, and leave the `Annotations` object unmodified (or create a new sorted one temporarily for plotting if it's easier).